presto-testing/src/main/java/io/prestosql/testing/DistributedQueryRunner.java

```diff
@@ -547,6 +553,7 @@ private static void closeUnchecked(AutoCloseable closeable)
         private Module additionalModule = EMPTY_MODULE;
         private Optional<Path> baseDataDir = Optional.empty();
         private List<SystemAccessControl> systemAccessControls = ImmutableList.of();
+        private List<Closeable> onClose = new ArrayList<>();
```

Comment:
Closer?

Reply:
This is a suggestion for a name?

Reply:
No, to use `Closer` from guava

Reply:
The Closer is eventually used in the DQR being built.